src/sentry/grouping/enhancer/parser.py

```diff
@@ -29,7 +29,7 @@
 actions          = action+
 action           = flag_action / var_action
 var_action       = _ var_name _ "=" _ ident
-var_name         = "max-frames" / "min-frames" / "invert-stacktrace" / "category"
```

Comment:
If any stack trace rules have it we can remove it after deployment.

The only way to invert the stack trace was via the hierarchical code which has already been removed:
https://github.com/getsentry/sentry/pull/77235/files#diff-765101e1faf45dddc42ee634ea872031228b3072cd6a28d41c533c3aba7414d0L491